jailer/src/env.rs

```diff
@@ -52,6 +52,7 @@ pub struct Env {
     seccomp_level: u32,
     start_time_us: u64,
     start_time_cpu_us: u64,
+    extra_args: Vec<String>,
```

Comment:
Nit: I would name this something like `forwarded_args`, but it's up to you.

Reply:
I like `extra_args` more.